<!-- -->
<launch>
  <arg name="pathDensity"           default="0.75" /> <!-- distance between each two waypoints-->
  <arg name="enableSmoothing"         default="true" /> <!-- 1 or 0 -->
  <arg name="enableLaneChange"         default="false" /> <!-- 1 or 0 -->
  <arg name="enableRvizInput"         default="false" /> <!-- 1 or 0 -->
  <arg name="enableReplan"            default="true" /> <!-- 1 or 0 -->  
  <arg name="velocitySource"          default="1" /> <!-- read velocities from (0- Odometry, 1- autoware current_velocities, 2- car_info) "" -->
  <arg name="mapSource"             default="0" /> <!-- Autoware=0, Vector Map Folder=1, kml file=2 -->
  <arg name="mapFileName"           default="" /> <!-- incase of kml map source -->
  <arg name="enableDynamicMapUpdate"       default="false" />  
  
<node pkg="op_global_planner" type="op_global_planner" name="op_global_planner" output="screen">
    
    <param name="pathDensity"         value="$(arg pathDensity)" />
    <param name="enableSmoothing"       value="$(arg enableSmoothing)" />
    <param name="enableLaneChange"       value="$(arg enableLaneChange)" />
    <param name="enableRvizInput"       value="$(arg enableRvizInput)" />
    <param name="enableReplan"         value="$(arg enableReplan)" />        
    <param name="velocitySource"       value="$(arg velocitySource)" />
    <param name="mapSource"         value="$(arg mapSource)" />
    <param name="mapFileName"         value="$(arg mapFileName)" />
    
    <param name="enableDynamicMapUpdate"   value="$(arg enableDynamicMapUpdate)" />
          
  </node> 
  
</launch>